### PR TITLE
[REVIEW] Upgrade to `arrow-11`

### PR DESCRIPTION
**Pull request by galipremsagar**
This PR upgrades arrow to `11.0` version.

xref: https://github.com/rapidsai/cudf/pull/12757

### CODE REVIEW

File: conda/recipes/versions.yaml [hunk 1]
```diff
@@ -99,7 +99,7 @@ jupyterlab_version:
 jupyter_packaging_version:
   - '>=0.7.0,<0.8'
 librdkafka_version:
-  - '=1.7.*'
+  - '>=1.9.0,<1.10.0a0"
```
Comment (galipremsagar) [inline conda/recipes/versions.yaml:102]:
```suggestion
  - '>=1.9.0,<1.10.0a0'
```

**Comment by ajschmidt8** [comment]
rerun tests

**Comment by ajschmidt8** [comment]
rerun tests

**Comment by ajschmidt8** [comment]
rerun tests

**Comment by ajschmidt8** [comment]
rerun tests

**Comment by ajschmidt8** [comment]
rerun tests

**Comment by ajschmidt8** [comment]
I think GitHub may be having some issues this morning which are preventing Jenkins from setting the statuses for these checks.

I will try rerunning the tests again later today.

**Review by msadang — APPROVED** [review]
Version change looks good.

**Comment by ajschmidt8** [comment]
rerun tests

**Comment by galipremsagar** [comment]
Alright, looks like the jobs worked this time.

**Pull request merged**